docs/examples/vault-support/test.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Update the Messaging Topology operator deployment, providing it with the Vault related environment variables
+echo "Updating Messaging Topology operator to work with Vault..."
+kubectl set env deployment/messaging-topology-operator -n rabbitmq-system --containers='manager' OPERATOR_VAULT_ROLE=messaging-topology-operator VAULT_ADDR=http://vault.default.svc.cluster.local:8200
```

Comment:
Minor issue: `VAUILT_ADDR` assumes we ran `setup.sh` from the `default` namespace.